test/test_contrib.py

```diff
@@ -415,9 +415,33 @@ def test_gradcheck(self, device, dtype):
 class TestImageStitcher:
     @pytest.mark.parametrize("estimator", ['ransac', 'vanilla'])
     def test_smoke(self, estimator, device, dtype):
-        B, C, H, W = 1, 3, 224, 224
-        sample1 = torch.rand(B, C, H, W, device=device, dtype=dtype)
-        sample2 = torch.rand(B, C, H, W, device=device, dtype=dtype)
+        B, C, H, W = 1, 3, 6, 6
```

Comment:
Continue to fail on macos https://github.com/kornia/kornia/actions/runs/3639450296/jobs/6142845358#step:4:9575